src/editors/sharedComponents/InsertLinkModal/BlocksList/index.jsx

```diff
@@ -0,0 +1,149 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Button, TransitionReplace, ActionRow } from '@edx/paragon';
+import { ArrowForwardIos, ArrowBack } from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import {
+  blockTypes,
+  getSectionsList,
+  getChildrenFromList,
+} from '../utils';
+
+import messages from './messages';
+import './index.scss';
+
+const BlocksList = ({ blocks, onBlockSelected }) => {
+  const intl = useIntl();
+  const messageBlockType = {
+    [blockTypes.section]: intl.formatMessage(
+      messages.blocksListSubsectionTitle,
+    ),
+    [blockTypes.subsection]: intl.formatMessage(messages.blocksListUnitTitle),
+    [blockTypes.unit]: intl.formatMessage(messages.blocksListUnitTitle),
+  };
+
+  const [blockState, setBlockState] = useState({
+    blockSelected: {},
+    type: blockTypes.subsection,
+    hasNavigated: false,
+    blocksNavigated: [],
+  });
+
+  const sections = getSectionsList(blocks);
+  const subSections = getChildrenFromList(
```

Comment:
```suggestion
  const subsections = getChildrenFromList(
```
^ Same suggestion for all the occurrences of `subSections`.

Reply:
I'm not sure about this suggestion, in Js the usual is use camelCase https://www.robinwieruch.de/javascript-naming-conventions/

I would recommend to keep the `subSections` name, or am I ignoring something?

Reply:
@01001110J You're right in JS it's better to use camelCase but "subsection" is only one world I think camelCase is usefull for two words or above, what do you think?

Reply:
As a personal recommendation I would say that it is better to use the camelCase, but the final decision is yours. :eyes: